docs/use/glue.md

```diff
@@ -11,16 +11,17 @@ kernelspec:
   name: python3
 ---
 
-(glue)=
+(glue/main)=
 
 # Insert variables into pages with `glue`
```

Comment:
It looks like the ideas around non-Python glue did not make it into this refactor, is that right? Did we hit a hard blocker there? Or is this still a thing to explore in the future?

Reply:
There's a way forward, as a drop in "companion" to Python glue (i.e. you could in principal use them both at the same time, so no deprecations needed), see https://github.com/jupyter/nbclient/pull/160#issuecomment-1005905731
I could essentially write it in to jupyter-cache right now, but obviously prefer it was part of nbclient. Plus ideally we would also introduce a new top-level `user_expressions` key in the nbformat JSON schema

Reply:
See https://github.com/executablebooks/meta/issues/681